<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>527</width>
                <height>259</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QWidget" name="horizontalLayoutWidget">
                <property name="geometry">
                    <rect>
                        <x>30</x>
                        <y>10</y>
                        <width>461</width>
                        <height>101</height>
                    </rect>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <widget class="QLabel" name="filename_label">
                            <property name="text">
                                <string>选择文件</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QPushButton" name="choose_button">
                            <property name="text">
                                <string>确定</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </widget>
            <widget class="QWidget" name="horizontalLayoutWidget_2">
                <property name="geometry">
                    <rect>
                        <x>89</x>
                        <y>120</y>
                        <width>341</width>
                        <height>80</height>
                    </rect>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                        <widget class="QPushButton" name="start_button">
                            <property name="text">
                                <string>开始</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="stop_button">
                            <property name="text">
                                <string>暂停</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="exit_button">
                            <property name="text">
                                <string>退出</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>527</width>
                    <height>26</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
